locales/en-US/app.ftl

```diff
@@ -993,6 +987,53 @@ TransformNavigator--collapse-function-subtree = Collapse subtree: { $item }
 #   $item (String) - Search filter of the markers that transform will apply to.
 TransformNavigator--drop-samples-outside-of-markers-matching = Drop samples outside of markers matching: “{ $item }”
 
+## "Dropped functions" settings UI
+##
+## A "dropped function" is a function which is excluded from the sample data.
+## More precisely, a sample will be excluded if its stack contains at least one
+## stack frame whose function is on the list of dropped functions. The excluded
+## samples are grayed out in the activity timeline, and they don't contribute to
+## the timings shown in the call tree and flame graph.
+## The user can add functions to the list of dropped functions by right-clicking
+## a row in the call tree and choosing "Drop samples with this function" in the
+## context menu.
+## To revert this action, they can open the panel of dropped functions and click
+## the "x" button next to the function that they no longer want to exclude.
+
+# The button which shows how many functions are currently on the list of dropped
+# functions. Clicking this button will open the "dropped functions" panel.
+#
+# Variables:
+#   $numberOfDroppedFunctions - The number of dropped functions, can be zero.
+DroppedFunctions--settings-button =
+    .label =
+        { $numberOfDroppedFunctions ->
+            [one] 1 dropped function
+        *[other] { $numberOfDroppedFunctions } dropped functions
```

Comment:
nit: fix the indentation. I would also shorten the variable name, e.g. just `$num`.